build.gradle

```diff
@@ -12,8 +12,8 @@ repositories {
 
 dependencies {
     testImplementation 'org.jetbrains.kotlin:kotlin-test:1.6.0'
-    implementation 'org.json:json:20171018'
```

Comment:
can you remind me why we don't need this anymore?

Reply:
I decided to utilize `Gson` which has proven a lot easier to use in order to serialize and deserialize objects to/from JSON